src/applications/proxy-rewrite/sass/consolidated-patches.scss

```diff
@@ -170,6 +170,21 @@ header.merger {
   }
 }
 
+#vetnav-va-benefits-and-health-care.vetnav-panel {
+  width: 250px !important;
+
+  @include media($medium-large-screen) {
+    &.vetnav-submenu--expanded {
+      width: 768px !important;
+    }
+  }
+  @include media($small-desktop-screen) {
+    &.vetnav-submenu--expanded {
+      width: 100% !important;
```

Comment:
Why is this `100%` but its counterpart in `src/platform/site-wide/sass/modules/_m-vet-nav.scss` is explicitly set to `1008px`? I notice it doesn't behave exactly the same (which is probably obvious). When this loads as an injected header on a legacy site, the content pane is exactly the same width as the navbar above it (due to `100%`).

![image](https://user-images.githubusercontent.com/6863534/192653452-e636c6d5-298d-485e-9f54-19d236a045c6.png)


But when this is loaded traditionally on the modernized site, the content pane is slightly wider than the nav bar above it:
![image](https://user-images.githubusercontent.com/6863534/192653603-99485c34-83e3-4af6-bc96-82d3aef8e6ff.png)

Reply:
I believe i left this in by mistake while trying to trouble shoot the menu width on injected header sites. Will remove and revert to 1008px to match modernized sites.